demos-europe/demosplan-core
# F 30480 custom default publication of documentcategories

**Ticket:** https://yaits.demos-deutschland.de/T30480

To cover different 
In order to reduce the effort on importing documents (ZIP-archive), it should be possible for the user to choose if all containing elements should be initially published or not.

### How to review/test
Import a ZIP-archive in the document section. After importing the files, you should see a checkbox on the overview where you can choose to publish the categories automatically or not. 
All contained elements of the imported zip-archive should be in the selected state.

### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
